### PR TITLE
Fix issues with new API changes

### PR DESCRIPTION
**Pull request by Soheab**
This PR adds the following attributes to blist.modules.Bot: `donate_url`, `avatar_hash`, `privacy_policy_url` and `status`. And fixes `blist.modules.Bot.add_date` not returning a proper datetime.datetime object.

### CODE REVIEW

File: blist/models.py [hunk 1]
```diff
@@ -21,9 +22,13 @@ def __init__(self, **kwargs):
         self.vanity_url: str = kwargs.get("vanity_url")
         self.server_count: int = kwargs.get("server_count")
         self.shard_count: int = kwargs.get("shard_count")
-        self.add_date: datetime.datetime = datetime.datetime.fromtimestamp(kwargs.get("add_date"))
+        self.joined: datetime.datetime = datetime.datetime.fromisoformat(kwargs.get("joined").strip("Z"))
```
Comment (joshuapatel) [inline blist/models.py]:
```suggestion
        self.add_date: datetime.datetime = datetime.datetime.fromisoformat(kwargs.get("joined").strip("Z"))
```
We shouldn't be breaking backward compatibility for our current users, though there are probably no current users except for you.

File: README.md [hunk 2]
```diff
@@ -133,9 +134,13 @@ Fetches information about a bot. Does not require authorization.
 |vanity_url|`str`|The vanity URL to the bot on the Blist website|
 |server_count|`int`|Amount of guilds the bot is in|
 |shard_count|`int`|Amount of shards the bot is using|
-|add_date|`datetime.datetime`|The date the bot was added to the Blist website|
+|joined|`datetime.datetime`|The date the bot was added to the Blist website|
```
Comment (joshuapatel) [inline README.md]:
```suggestion
|add_date|`datetime.datetime`|The date the bot was added to the Blist website|
```
Change this back for the sake of backward compatibility.

File: README.md [hunk 3]
```diff
@@ -133,9 +134,13 @@ Fetches information about a bot. Does not require authorization.
...
 |invites|`int`|Amount of times the bot has been invited from the Blist website|
 |page_views|`int`|Amount of views the bot page has received|
+|donate_url|`str`|Link to donate|
```
Comment (joshuapatel) [inline README.md]:
```suggestion
|donate_url|`str`|Link to the bot's donate page|
```

File: README.md [hunk 4]
```diff
@@ -133,9 +134,13 @@ Fetches information about a bot. Does not require authorization.
...
+|avatar_hash|`str`|The bot's avatar's hash|
```
Comment (joshuapatel) [inline README.md]:
```suggestion
|avatar_hash|`str`|The bot avatar hash|
```

File: README.md [hunk 5]
```diff
@@ -133,9 +134,13 @@ Fetches information about a bot. Does not require authorization.
...
+|privacy_policy_url|`str`|Link to bot's privacy url|
+|status|`str`|The bot's current status.|
```
Comment (joshuapatel) [inline README.md]:
```suggestion
|status|`str`|The bot's current status|
```

File: README.md [hunk 6]
```diff
@@ -133,9 +134,13 @@ Fetches information about a bot. Does not require authorization.
 |vanity_url|`str`|The vanity URL to the bot on the Blist website|
 |server_count|`int`|Amount of guilds the bot is in|
 |shard_count|`int`|Amount of shards the bot is using|
-|add_date|`datetime.datetime`|The date the bot was added to the Blist website|
+|joined|`datetime.datetime`|The date the bot was added to the Blist website|
 |invites|`int`|Amount of times the bot has been invited from the Blist website|
 |page_views|`int`|Amount of views the bot page has received|
+|donate_url|`str`|Link to donate|
+|avatar_hash|`str`|The bot's avatar's hash|
+|privacy_policy_url|`str`|Link to bot's privacy url|
```
Comment (joshuapatel) [inline README.md]:
```suggestion
|privacy_policy_url|`str`|Link to the bot's privacy policy URL|
```

**Review by joshuapatel — CHANGES_REQUESTED** [review]
Overall, this PR looks quite nice, but there are a few issues that need to be addressed, which I have outlined in this review.

File: blist/models.py [hunk 7]
```diff
@@ -60,4 +65,4 @@ def __init__(self, **kwargs):
         self.votes: typing.List[Vote] = [Vote(**vote) for vote in kwargs.get("votes")]
 
     def __repr__(self):
-        return f"<{self.__class__.__name__} monthly={self.monthly} total={self.total} votes={self.votes}>"
\ No newline at end of file
+        return f"<{self.__class__.__name__} monthly={self.monthly} total={self.total} votes={self.votes}>"
```
Comment (joshuapatel) [inline blist/models.py]: Please remove the extra newline added at the end of the file.

**Review by joshuapatel — CHANGES_REQUESTED** [review]
Thanks for making the changes requested! All I ask is that you fix the issue below and then I'll merge this PR.

File: README.md [hunk 8]
```diff
@@ -136,6 +137,10 @@ Fetches information about a bot. Does not require authorization.
 |add_date|`datetime.datetime`|The date the bot was added to the Blist website|
 |invites|`int`|Amount of times the bot has been invited from the Blist website|
 |page_views|`int`|Amount of views the bot page has received|
+|donate_url|`str`||Link to the bot's donate page|
+|avatar_hash|`str`|The bot avatar hash|
```
Comment (GavynStanley) [inline README.md]: Shouldn't this be `bot's` ?

Comment (joshuapatel) [reply README.md]: Yes, it should be. @Soheab ~~fix this~~

File: README.md [hunk 9]
```diff
@@ -123,10 +123,11 @@ Fetches information about a bot. Does not require authorization.
 |website|`str`|Link to the bot's website|
 |github|`str`|Link to the bot's GitHub repository|
 |short_description|`str`|Short description of the bot|
+|long_description|`str`|Long description of the bot|
 |prefix|`str`|The bot's prefix|
 |invite_url|`str`|Link to the invite URL of the bot|
-|support_server|`str`|Bot support guild invite code|
-|tags|List[`str`]|List of bot tags on the Blist website|
+|support_server|`str`|Bot's support guild code slug|
```
Comment (joshuapatel) [inline README.md]: This sounds straight-up confusing. I would rephrase it to something else. If you cannot come up with a phrase that sounds better, I'm fine merging as-is.

Comment (GavynStanley) [reply README.md]: `Bot's support server invite code`

Comment (Soheab) [reply README.md]: server's* ?

Comment (joshuapatel) [reply README.md]: `guild` is used instead of `server` to remove any ambiguity associated with a Discord guild and an actual server.

Comment (joshuapatel) [reply README.md]:
```suggestion
|support_server|`str`|The bot's support guild invite code|
```
How about this?

Comment (Soheab) [reply README.md]: Looks good to me.

File: README.md [hunk 10]
```diff
@@ -123,10 +123,11 @@ Fetches information about a bot. Does not require authorization.
 |website|`str`|Link to the bot's website|
 |github|`str`|Link to the bot's GitHub repository|
 |short_description|`str`|Short description of the bot|
+|long_description|`str`|Long description of the bot|
 |prefix|`str`|The bot's prefix|
 |invite_url|`str`|Link to the invite URL of the bot|
-|support_server|`str`|Bot support guild invite code|
-|tags|List[`str`]|List of bot tags on the Blist website|
+|support_server|`str`|The bot's support guild invite code|
+|tags|List[`str`]|List of the bot's tag on the Blist website|
```
Comment (joshuapatel) [inline README.md]:
```suggestion
|tags|List[`str`]|List of the bot's tags on the Blist website|
```

**Review by joshuapatel — CHANGES_REQUESTED** [review]

**Review by joshuapatel — APPROVED** [review]
Looks great!

**Comment by Soheab** [comment]
> Looks great!

Thank you!
